tools/bazar/fields/ListField.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace YesWiki\Bazar\Field;
+
+use Psr\Container\ContainerInterface;
+
+abstract class ListField extends BazarField
+{
+    public function __construct(array $values, ContainerInterface $services)
+    {
+        parent::__construct($values, $services);
+
+        $this->recordId = $values[self::FIELD_TYPE] . $values[self::FIELD_ID] . $values[6];
```

Comment:
```suggestion
        $this->recordId = $values[self::FIELD_TYPE] . $values[self::FIELD_ID] . $values[self::FIELD_LIST_LABEL];
```
idem pour rester cohérent avec mon autre commentaire